content/docs/install/helm-install/index.md

```diff
@@ -317,7 +317,7 @@ For example, if a user sets `prefixPath=/mykeptn` in the Helm install/upgrade co
 the Keptn API is located under `http://HOSTNAME/mykeptn/api` and the Keptn Bridge is located under `http://HOSTNAME/mykeptn/bridge`:
 
 ```
-helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=apiGatewayNginx.type=LoadBalancer,continuousDelivery.enabled=true,prefixPath=/mykeptn
+helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=apiGatewayNginx.type=LoadBalancer,bridge.installationType=CONTINUOUS_DELIVERY,prefixPath=/mykeptn
```

Comment:
This setting is actually in the default values anyways, so there's no need to set anything here. We can just remove that `--set`
```suggestion
helm upgrade keptn keptn --install -n keptn --create-namespace --wait --version=0.19.3 --repo=https://charts.keptn.sh --set=apiGatewayNginx.type=LoadBalancer,prefixPath=/mykeptn
```

Reply:
apologies for late response, I have addressed this and pushed it, please review again.